### ISSUE TITLE
periodic error from o.validate

### ISSUE THREAD

**Issue by equilet** [CNMAT/CNMAT-MMJSS#27]
`o.validate: received something that is neither an OSC bundle nor a message`

I get this every once in a while, and I'm not sure how to reproduce it.
I'll keep my eye out.

**Comment by equilet** [comment]
seems like opening up the following files in succession (switching between them) can yield it:

`/extras/MMJSS-extras/solutions/01_monday_io.maxpat`
`/extras/MMJSS-extras/solutions/02_monday_diamond_tones.maxpat`

**Comment by ilzxc** [comment]
This should only be happening when you have an esplora plugged in -- we
looked into this last Fall, when we were using Esploras in Music158. If I
recall correctly, the issue stems from slip-encoded packets arriving out of
order. In such a case, the packet is indeed malformed.

**Comment by equilet** [comment]
Yes, I can verify that it only occurs with the esplora plugged in.
